packages/main/src/components/add-prefix-modal/utils/Common.jsx

```diff
@@ -0,0 +1,120 @@
+import { authAxios } from "./Api";
+import styles from "./styles/PaymentMethod.module.css";
+import mastercard from "../assets/Mastercard.svg";
+import jcb from "../assets/jcb.svg";
+import visa from "../assets/visa.svg";
+import amex from "../assets/american-express.svg";
+import dinner from "../assets/dinners-club.svg";
+import discover from "../assets/discover.svg";
+
+// return the token from the session storage
+export const getToken = () => {
+  return sessionStorage.getItem("token") || null;
+};
+
+export const getUser = () => {
+  const result = JSON.parse(sessionStorage.getItem("user") || null);
+  return result;
+};
+
+// get Current Workspace
+export const getCurrentWorkspace = () => {
+  const currentWorkspace = localStorage.getItem("currentWorkspace") || null;
+  return currentWorkspace;
+};
+
+// get Current Workspace Data
+export const getCurrentWorkspaceData = () => {
+  if (getCurrentWorkspace()) {
+    authAxios
+      .get(`/organizations/${getCurrentWorkspace()}`)
+      .then(res => {
+        return res.data.data;
+      })
+      .catch(err => {
+        console.error(err);
+      });
+  }
+};
+
+export const ValidateCard = (name, style) => {
```

Comment:
the app is free so cards are not necessary

Reply:
Removed since its not been used in the component